src/referentielijsten/api/admin.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib import admin
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from .admin_list_filters import GeldigListFilter
+from .models import Item, Tabel
+
+
+@admin.register(Item)
+class ItemAdmin(admin.ModelAdmin):
+    list_display = ("tabel", "code", "naam", "is_geldig")
+    list_filter = (GeldigListFilter, "tabel__code")
```

Comment:
Now the filters look like this 
![image](https://github.com/maykinmedia/referentielijsten/assets/47745906/0f4a215d-78b7-494a-ac1a-6fe6ec550327)

It can be confusing, because `Item` model also has `code` field. Please change the title of `table__code` filter.